app/src/components/home/Home.test.js

```diff
@@ -76,10 +77,12 @@ it('tries to submit an empty query', () => {
   expect(nowStore.query.current).toEqual(SEARCH_STR);
   expect(nowStore.query.submitted).toEqual(false);
   expect(nowStore.query.isWaiting).toEqual(false);
+  expect(nowStore.notify.msg).toEqual(NOTIFY_MSG);
```

Comment:
great, I like this type of test... carryout a few user interactions and then verify all the appropriate store changes (which also confirms correct component, action and reducer functionality along the way)